bridges/scalajs-1/src/main/scala/bloop/scalajs/JsBridge.scala

```diff
@@ -151,11 +144,12 @@ object JsBridge {
     if (nodeModules.toFile().exists()) {
       logger.debug("Node.js module path: " + nodeModules.toString())
       val fullEnv = Map("NODE_PATH" -> nodeModules.toString()) ++ env
-      val config =
-        NodeJSConfig().withExecutable(nodePath).withCwd(Some(baseDirectory)).withEnv(fullEnv)
       val nodeEnv =
-        if (!jsConfig.jsdom.contains(true)) new NodeJSEnv(logger, config)
-        else new JsDomNodeJsEnv(logger, config)
+        if (!jsConfig.jsdom.contains(true))
```

Comment:
I know you didn't change this logic, but is this actually needed seeing that it will always be `false`?

Reply:
Is it? If it's empty it will be false and also if it contains false the flag will be true. Or am I not seeing it?
```scala
!Some(false).contains(true) == true
!Some(true).contains(true) == false
!None.contains(true) == true
```

Reply:
Ahhh, no you're right, sorry I read this too quickly and didn't understand it.... ignore me.